portfolio/src/main/webapp/script.js

```diff
@@ -53,6 +53,17 @@ function createCommentElement(comment) {
 
   commentElement.appendChild(nameElement);
   commentElement.appendChild(textElement);
+
+  imageURL = comment.imageURL;
+  if(imageURL != null){
+    console.log("Vibes");
```

Comment:
Consider using more descriptive logging if you are going to keep this in production code.

Reply:
Thanks Kenneth. I don't need it in my production code, so I'll remove it.